tests/functional/Helmich/TypoScriptLint/Tests/Linter/Fixtures/ConfigNoCache/output.txt

```diff
@@ -0,0 +1,2 @@
+2;;Setting config.no_cache = 1 is discouraged as it is bad for performance.Consider using USER_INT object instead. Found in path: testPage2.config.no_cache;warning;Helmich\TypoScriptLint\Linter\Sniff\ConfigNoCacheSniff
+8;;Setting config.no_cache = 1 is discouraged as it is bad for performance.Consider using USER_INT object instead. Found in path: testPage.config.no_cache;warning;Helmich\TypoScriptLint\Linter\Sniff\ConfigNoCacheSniff
```

Comment:
:+1: for thinking about the test cases!

Reply:
Thank you for making the code  easy to read and extend!
I was surprised how easy if was to add new sniff despite no docs  and no previous experience in this from my side :)